includes/class-sensei-course.php

```diff
@@ -3346,11 +3346,13 @@ public static function prerequisite_complete_message() {
 	 * @param WP_Post $course The Course.
 	 */
 	public function log_initial_publish_event( $course ) {
-		$product_id       = get_post_meta( $course->ID, '_course_woocommerce_product', true );
+		$product_ids = get_post_meta( $course->ID, '_course_woocommerce_product', false );
+		$product_count = empty( $product_ids ) ? 0 : count( $product_ids );
```

Comment:
We could probably do this right inline and eliminate the need for creating a variable.